Source/WebCore/platform/graphics/gstreamer/ImageDecoderGStreamer.cpp

```diff
@@ -228,6 +228,16 @@ const ImageDecoderGStreamerSample* ImageDecoderGStreamer::sampleAtIndex(size_t i
     return toSample(iter);
 }
 
+ImageDecoderGStreamer::InnerDecoder::~InnerDecoder()
+{
+    GST_DEBUG_OBJECT(m_pipeline.get(), "Destructing decoder");
```

Comment:
I would do this a trace, but lgtm as well

Reply:
imho trace is for noisy logs, appearing with high frequency, i don't think it applies here :)